build-logic/jvm/src/main/kotlin/gradlebuild/propagated-env-variables.kt

```diff
@@ -46,6 +46,7 @@ val propagatedEnvironmentVariables = listOf(
 
     // Used by Gradle test infrastructure
     "REPO_MIRROR_URLS",
+    "YARNPKG_MIRROR_URL",
```

Comment:
Why do we propagate this environment variable? For the tests which build Gradle itself?

Reply:
Yes, I would like them to have this configured as well.